checks.d/mcache.py

```diff
@@ -77,7 +79,8 @@ class Memcache(AgentCheck):
         "curr_connections",
         "connection_structures",
         "threads",
-        "pointer_size"
+        "pointer_size",
+        "listen_disabled_num"
```

Comment:
as far as I understand, `listen_disabled_num` is an uncapped counter. Reporting values that could potentially grow to infinity is not good practice. It would be more relevant to collect it as a rate (difference between the previous value and the current one). Rates also allow you to draw this kind of graphs:

![screen shot 2015-10-23 at 1 59 15 pm](https://cloud.githubusercontent.com/assets/5287287/10692008/5ff5b632-798e-11e5-9596-ced22560752d.png)

Instead of a simple line growing slowly:
![image](https://cloud.githubusercontent.com/assets/5287287/10692081/080e0c02-798f-11e5-9eec-81d67ee1e23a.png)

It also makes alerting easier (setting a threshold value wouldn't be as effective on a gauge as on a rate).